components/dash-core-components/src/components/RadioItems.react.js

```diff
@@ -63,39 +74,48 @@ export default class RadioItems extends Component {
 }
 
 RadioItems.propTypes = {
+    /**
+     * An array of options, or inline dictionary of options
+     */
+    options: PropTypes.oneOfType([
```

Comment:
Let's support the same 3 sets of option types across RadioItems, Dropdown, CheckList, and DataTable.columns for consistency:
- List of dictionaries (as before)
- Dictionary (value: label)
- List of numbers, strings, or bools

Reply:
RadioItems, Dropdown, CheckList updated in 63ced025

Reply:
Great. Let's skip DataTable for now. We can always do it later.

Reply:
f990b43c